src/System.Diagnostics.Tools/tests/System/Diagnostics/CodeAnalysis/SuppressMessageAttributeTests.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace System.Diagnostics.CodeAnalysis.Tests
+{
+    public static class SuppressMessageAttributeTests
+    {
+        [Fact]
+        public static void TestConstructorBasic()
+        {
+            string expectedCategory = "Category";
+            string expectedCheckId = "CheckId";
+            SuppressMessageAttribute sma = new SuppressMessageAttribute(expectedCategory, expectedCheckId);
+            Assert.Equal(expectedCategory, sma.Category);
+            Assert.Equal(expectedCheckId, sma.CheckId);
+        }
+
+        [Fact]
+        public static void TestConstructorExtended()
```

Comment:
I think a rename of the method is fine, given that these tests are mostly just for code coverage. There's nothing to really test on this class as it's essentially just a container for the properties here. The other option is to test each property individually in different test cases.